src/assets/aerie-phoenix-wordmark.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="275" height="39" viewBox="0 0 275 39" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Shouldn't this image be used in the editor instead of the text "Phoenix Editor"?